### PR TITLE
solution

### PR DESCRIPTION
**Pull request by nrjss67**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,22 @@
...
+
+
+def move_file(command: str) -> None:
+    command_split = command.split()
+    if len(command_split) == 3 and command_split[0] == "mv":
+        comm, file_name, path = command_split
+
+    if "/" not in path:
+        os.rename(file_name, path)
+        return
```
Comment (vsmutok) [inline app/main.py:11]: it's better to raise some error in this case so that the user knows if something goes wrong

File: app/main.py [hunk 2]
```diff
@@ -1 +1,22 @@
...
+
+    if path.endswith("/"):
+        os.makedirs(path, exist_ok=True)
+        os.replace(file_name, path)
+        return
```
Comment (vsmutok) [inline app/main.py:16]: I think this is redundant

File: app/main.py [hunk 3]
```diff
@@ -1 +1,22 @@
...
+
+    if path.endswith(".txt"):
+        new_file_name = path.split("/")[-1]
+        path_without_fn = "/".join(path.split("/")[:-1])
+        os.makedirs(path_without_fn, exist_ok=True)
+        os.replace(file_name, f"{path_without_fn}/{new_file_name}")
```
Comment (vsmutok) [inline app/main.py:22]: why `.txt` ?

What if we need move other type of file? main.py for example

**Review by vsmutok — CHANGES_REQUESTED** [review]

**Review by vsmutok — CHANGES_REQUESTED** [review]
.

**Review by vasylhnatiuk — APPROVED** [review]

**Pull request closed without merging**